parser.y

```diff
@@ -4580,6 +4582,10 @@ SumExpr:
 	{
 		$$ = &ast.AggregateFuncExpr{F: $1, Args: []ast.ExprNode{$4}, Distinct: $3.(bool)}
 	}
+|   builtinJsonObjectAgg '(' Expression ',' Expression ')'
```

Comment:
1. Please use single tab rather than several spaces, just like the other builtin aggregate functions:

https://github.com/pingcap/parser/blob/169e7384f4cf04d92675bf65784974de77f69f95/parser.y#L4581

2. Please notice that the `BuggyDefaultFalseDistinctOpt` before `Expression`, which allows optional `ALL` token before the expression, for example:

```
SELECT JSON_OBJECTAGG(all c, all i) FROM t;
```
is valid and is equivalent to:
```
SELECT JSON_OBJECTAGG(c, i) FROM t;
```
3. It's better to add `OptWindowingClause` as well since window function is now supported in TiDB